parser/src/parser/tests/arithmetic_ops.rs

```diff
@@ -225,15 +225,6 @@ fn err_closing_paren_without_opening_paren() {
     build_parse_test!(source).expect_unrecognized_token();
 }
 
-#[test]
-fn non_const_exp_fail() {
-    // Should fail if the exponent is not a constant
-    let source = "
-    integrity_constraints:
-        enf clk'^a = 1";
-    build_parse_test!(source).expect_unrecognized_token();
-}
-
```

Comment:
Instead of deleting this, let's replace it with a test for the non-const exponent

Reply:
Thanks. Done.